field_audit/models.py

```diff
@@ -703,7 +703,10 @@ def bulk_create(self, objs, *, audit_action=AuditAction.RAISE, **kw):
             return super().bulk_create(objs, **kw)
         assert audit_action is AuditAction.AUDIT, audit_action
 
-        from .field_audit import request
+        from .field_audit import is_audit_enabled, request
```

Comment:
What do you think of moving `is_audit_enabled` and related decorators to a different module (other than `field_audit/field_audit.py`) so they can be imported at the top of this file? I don't think they have any dependencies that would cause a circular import or otherwise require local import.